ui/src/styles.scss

```diff
@@ -312,11 +312,11 @@ body {
 }
```

Comment:
Include the generated ng build files from ui/src/dist folder for the pr changes

Reply:
dist folder updated.